dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/GistPagerControllerTest.java

```diff
@@ -42,9 +43,12 @@
 class GistPagerControllerTest extends AbstractGistControllerTest {
   @Test
   void testPager_Total_ResultBased() {
-    JsonObject gist =
-        GET("/users/{uid}/userGroups/gist?fields=name,users&total=true", getSuperuserUid())
-            .content();
+    String baseUrl = "/users/{uid}/userGroups/gist?fields=name,users";
+    JsonObject gist = GET(baseUrl + "&total=true", getSuperuserUid()).content();
+    assertHasPager(gist, 1, 50, 1);
+    gist = GET(baseUrl + "&totalPages=true", getSuperuserUid()).content();
+    assertHasPager(gist, 1, 50, 1);
+    gist = GET(baseUrl + "&totalPages=true&total=false", getSuperuserUid()).content();
```

Comment:
In tracker we opted to reject contradictory parameters as its not clear what the user wants 🤔

Reply:
:+1: will follow that